connector/tests/build.gradle.kts

```
@@ -13,7 +18,7 @@ dependencies {
     testImplementation(libs.testcontainers.postgresql)
     testImplementation(libs.testcontainers.hashicorp.vault)
     testImplementation(libs.testcontainers.keycloak)
+    testFixturesImplementation(libs.postgres)
```

```suggestion
    testImplementation(libs.postgres)
```

test fixtures are a particular gradle source set we're not using here, if I'm correct